src/internal/HtmlColor.h

```diff
@@ -76,6 +90,23 @@ struct HtmlColor
     };
 
     // ------------------------------------------------------------------------
+    // Parse a HTML4/CSS3 color name
+    //
+    // name - the color name
+    //
+    // It accepts all standard HTML4 names and, if USE_CSS3_COLORS macro is
+    // defined, the color names defined in CSS3 also
+    // ------------------------------------------------------------------------
+    bool Parse(const char* name);
```

Comment:
what if the name is stored in PROGMEM, how would a consumer of the API get it parsed?
Parse_P()?